updatecli/updatecli.d/openssh.yml

```
@@ -0,0 +1,54 @@
+name: Bump `openssh` version on Windows
+
+scms:
+  default:
+    kind: github
+    spec:
+      user: "{{ .github.user }}"
+      email: "{{ .github.email }}"
+      owner: "{{ .github.owner }}"
+      repository: "{{ .github.repository }}"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+      branch: "{{ .github.branch }}"
+
+sources:
+  lastVersion:
+    kind: githubrelease
+    name: Get the latest `openssh` version
+    spec:
+      owner: PowerShell
+      repository: Win32-OpenSSH
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+
+targets:
+  setGitLfsVersionWindowsNanoserver:
+    name: Update the `openssh` Windows version for Windows Nanoserver
```

```suggestion
    name: Update the OpenSSH Windows version for Windows Nanoserver
```

(nit) There is no CLI or code named `openssh`. Cased version is the official name as per https://www.openssh.com/